src/main/java/com/enterprise/fundraisermanager/controller/FundraiserManagerController.java

```diff
@@ -1,97 +1,125 @@
-        return fundraiserService.fetchAll();
-    }
-
-    @GetMapping("/fundraiser/{id}/")
-    public ResponseEntity fetchFundraiserByID(@PathVariable("id") Integer id) {
-        Fundraiser foundFundraiser = fundraiserService.fetchById(id);
-        HttpHeaders headers = new HttpHeaders();
-        headers.setContentType(MediaType.APPLICATION_JSON);
-        return new ResponseEntity(foundFundraiser, headers, HttpStatus.OK);
-    }
-
-    @PostMapping(value="/fundraiser", consumes="application/json", produces = "application/json")
-    public Fundraiser createFundraiser(@RequestBody Fundraiser fundraiser) {
-        Fundraiser newFundraiser = null;
-        try {
-            newFundraiser = fundraiserService.save(fundraiser);
-        }catch (Exception e) {
-            //TODO add logging
-        }
-        return newFundraiser;
-    }
-
-    @DeleteMapping("/fundraiser/{id}/")
-    public ResponseEntity deleteFundraiser(@PathVariable("id") Integer id) {
-        try {
-            fundraiserService.delete(id);
-            return new ResponseEntity(HttpStatus.OK);
-        } catch (Exception e) {
-            //TODO confirm correct response in catch, or modify it to be correct response.
-            return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
-        }
-    }
-
-    @GetMapping("/fundraisers")
-    public ResponseEntity searchFundraisers(@RequestParam(value = "searchTerm", required = false, defaultValue = "None") String searchTerm) {
-        String newSearchTerm = searchTerm + " ";
-        return new ResponseEntity(HttpStatus.OK);
-    }
-
-    /**
-     * Handle the members endpoint and return the members page.
-     * @return members page
-     */
-    @RequestMapping("/members")
-    public String members(){
-
-        return "members";
-    }
+package com.enterprise.fundraisermanager.controller;
+
+import com.enterprise.fundraisermanager.dto.Fundraiser;
+import com.enterprise.fundraisermanager.service.IFundraiserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.List;
+/**
+ * This class is in charge of sending all incoming requests to the appropriate service and method. The controller sends the response back to the requester once the response data is ready.
+ * The Fundraiser Management Application includes features such as displaying all fundraisers, adding new fundraisers, changing fundraiser details, removing fundraisers, and searching for fundraisers.
+ * @author .
+ */
+@Controller
+public class FundraiserManagerController {
```

Comment:
I made a controller package and placed the FundraiserManagerController.java in it. While Java doesn't impose any project structure, it's usually a good idea to arrange our source files, tests, configurations, data, and other code artifacts in a consistent manner. If your application grows in size (which it may in your case), a good project structure will help you manage the code and distribute the work among your team.

This is a helpful link: https://www.baeldung.com/java-clean-code